tests/chem.conf

```diff
@@ -0,0 +1,11 @@
+COMPILE | APP=ATM SUITES=FV3_GFS_v16,FV3_GFS_v16_RRTMGP,FV3_GFS_v16_gsd_chem                                                      |                                         | fv3 |
```

Comment:
Please move those into `rt_ccpp_dev.conf` - if not now, then by the time the code goes back to the authoritative repositories. Or we need to be prepared to maintain yet another file that only exists in this repo.